packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -263,19 +281,51 @@ export default class Meetings extends WebexPlugin {
         StaticConfig.set(this.config);
         LoggerConfig.set(this.config.logging);
         LoggerProxy.set(this.webex.logger);
-        if (this.webex.canAuthorize) {
```

Comment:
Do we need to add this check back  ?

Reply:
No, since we aren't doing anything that requires an authorized user on plugin init now.